components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -72,8 +72,9 @@ class AddonManager(
 
             // Get all available/supported addons from provider and add state if installed.
             // NB: We're keeping translations only for the default locale.
-            val locales = listOf(Locale.getDefault().language)
-            val supportedAddons = addonsProvider.getAvailableAddons(allowCache)
+            val userLanguage = Locale.getDefault().language
```

Comment:
I was wondering since we last spoke: Shouldn't we include our `LocaleManager` here as well e.g. Fenix uses it when switching language within the app via the `LocaleSettingsFragment`?

Reply:
Ok I will take a look.

Reply:
We have been using `Locale.getDefault().language` since we introduced add-ons without issues, let me take a look what will be the advantage pros and cons of using  `LocaleManager`.

Reply:
`LocaleManager` is external dependency if we use it internally we will be assuming that consumers will always be using it? Should it be better to just add a language parameter to `getAddons`?

Reply:
And let consumer decide in which language the add-ons should be shown?

Reply:
Yes, probably best. This was just a thought though. We don't need to solve it as part of this PR.

Reply:
👍🏽